### ISSUE TITLE
Clean up community-engineering OpsGenie on-call and merge into Arch-BOM

### ISSUE THREAD

**Issue by timmc-edx** [edx/edx-arch-experiments#133]
Possibly blocked until Arbi-BOM and FED-BOM are set up in OpsGenie

- [x] New Relic alert policies
  - [x] prod-frontend-app-account-community-engineering https://onenr.io/0oR8AaAvejG
    - Blocked on https://github.com/edx/edx-arch-experiments/issues/118
  - [x] prod-edx-edxapp-lms-community-engineering https://onenr.io/07j90x0VxRO
    - Only alerts when `code_owner_squad = 'community-engineering'`
    - `select count(*) from PageView where code_owner_squad = 'community-engineering' since 1 month ago timeseries` (and same for `from Transaction`) shows zero for last two weeks, and squad is no longer present in config, so can just delete this policy.
  - [x] prod-edge-edxapp-lms-community-engineering https://onenr.io/0vwBpLpv9jp
    - Same as above
- [x] New Relic "Community Engineering - Alerts" alerts classic channel: https://one.newrelic.com/nrai/alerts-classic/channels?account=88178&state=e68e7ecc-20fd-4e9e-f94a-54c17f699e33 (once policies are removed or re-routed)
- [x] Opsgenie CENG NR integration: https://2u-internal.app.opsgenie.com/settings/integration/edit/NewRelicV2/bbe7fe10-4ffa-407f-b9a4-5ea29e8d69fc?teamId=ff1fec59-f3d1-4b96-9534-52b5d0298bad (once NR alert policies are removed)
- [x] Opsgenie CENG team: https://2u-internal.app.opsgenie.com/teams/dashboard/ff1fec59-f3d1-4b96-9534-52b5d0298bad/main (once integrations are removed)

**Comment by robrap** [comment]
@rgraber: I'm leaving this as blocked until https://github.com/edx/edx-arch-experiments/issues/118 is completed. Once that is done, you are welcome to wrap up this ticket and/or pass it on to a future on-call. Thanks.

**Comment by rgraber** [comment]
Changed New Relic "Community Engineering - Alerts" alerts classic channel to point to the new FED-BOM team in OpsGenie.

**Comment by rgraber** [comment]
Renamed prod-frontend-app-account-community-engineering to prod-frontend-app-account-fed-bom

**Comment by robrap** [comment]
1. Are we updating notification channels for the policy as well? Note - from the old notification channel you could view all related policies to see if we missed anything.
2. We could also drop the notification channel once it is no longer used.
3. Does fed-BOM want an alert channel? I guess we could always just note that they could set that up if they wish.

**Comment by rgraber** [comment]
1. Yes, the notification channels were updated to remove arch-bom slack and point to the FED BOM team in OpsGenie
2. chose to rename instead of drop
3. We have left that to them and will help if they ask

**Comment by robrap** [comment]
@rgraber: The one specific email examples I remember was on one of the failing Jenkins jobs for translations of either the header or footer component. However, it may have just gone to a group email, and not to Opsgenie.

**Comment by robrap** [comment]
Thanks. I was misremembering. Yes - this was TNL. I can't remember where I thought I saw a CE email, and it was probably community-engineering at edx.org anyway.
